eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/actions/ShareActionExecutable.java

```diff
@@ -60,10 +62,24 @@ public void execute(@NonNull Context context)
 
         String eventText = eventText(context);
 
-        Intent sendIntent = new Intent();
-        sendIntent.setAction(Intent.ACTION_SEND);
-        sendIntent.putExtra(Intent.EXTRA_TEXT, eventText);
-        sendIntent.setType("text/plain");
+        Intent sendIntent;
+        if (context instanceof Activity)
```

Comment:
Yes, we should change the parameter to `Activity`. At present, I don't think that we'll use ActionExecutables from any other Context.

I think `ShareCompat.IntentBuilder` can return the "pureIntent" as well, so you still can use it.